R/pkg/inst/worker/worker.R

```diff
@@ -27,6 +27,58 @@ elapsedSecs <- function() {
   proc.time()[3]
 }
 
+computeHelper <- function(mode, partition, serializer, deserializer, key,
```

Comment:
propose to separate this function to two functions: compute() and outputResult(). Compute elapsed time and output elapsed time can be calculated within these two functions.
Something like follows:
```
  computeElapsedTime <- 0
  outputElapsedTime <- 0
  for (group in groups) {
    (output, elapsed time) <- compute(group)
    computeElapsedTime += elapsed time
    outputElapsedTime += outputResult(output)
  }
```